docs/source/cluster_management/clis.rst

```diff
@@ -98,21 +98,30 @@ Read or recalculate the immutable timestamp. Run ``./bin/atlasdb help timestamp`
 migrate
 -------
 
+.. danger::
+
+   Ensure you have backups before attempting a KVS migration.
+   Double check that `migrateConfig` (the target KVS) is specified correctly, since all tables in that KVS will be dropped.
+
 This CLI can help you migrate your AtlasDB client product from one KVS to another.
+The source KVS (which contains the data to migrate) should be specified in the `fromConfig`.
+The target KVS (which we will migrate to) should be specified in the `migrateConfig`.
 You will need to supply two different KVS configurations to the script.
-In the general case you first call ``–-setup``, then ``-–migrate``, then ``-–validate`` each time supplying the old and new configurations.
+In the general case you first call ``--setup``, which creates the necessary tables in the target KVS.
+Then, you call ``--migrate``, which writes the data. If this step fails, it can safely be resumed (with the same arguments).
+The CLI will check and skip past tables that have already been processed.
+Finally, you should call ``--validate`` to validate the migration.
 
-We currently only support doing KVS migrations offline (using the ``--offline`` flag), so you must shut down your AtlasDB backed service to perform the migration.
+It is highly recommended that KVS migrations are run offline (using the ``--offline`` flag), so you should shut down your AtlasDB backed service to perform the migration.
 For more information run ``./bin/atlasdb help migrate`` for more information.
+The typical usage is given below:
 
 .. code-block:: bash
 
-     ./bin/atlasdb-cli --offline migrate –-fromConfig from.yml --migrateConfig to.yml –-setup
-     ./bin/atlasdb-cli --offline migrate –-fromConfig from.yml --migrateConfig to.yml --migrate
-     ./bin/atlasdb-cli --offline migrate –-fromConfig from.yml --migrateConfig to.yml --validate
+     ./bin/atlasdb-cli --offline migrate --fromConfig from.yml --migrateConfig to.yml --setup
+     ./bin/atlasdb-cli --offline migrate --fromConfig from.yml --migrateConfig to.yml --migrate
+     ./bin/atlasdb-cli --offline migrate --fromConfig from.yml --migrateConfig to.yml --validate
 
```

Comment:
Should we add more details so that some of the technical details are clearer, e.g. which tables are/aren't migrated and why?